api/src/main/scala/dcos/metronome/api/v1/controllers/ApplicationController.scala

```diff
@@ -19,6 +21,18 @@ class ApplicationController(metrics: Metrics) extends RestController {
 
   def ping = Action { Ok("pong") }
 
+  def info = Action {
+
+    val infoJson = Json.toJson(
+      Map(
+        "version" -> MetronomeBuildInfo.version,
+        "lib-version" -> MetronomeBuildInfo.marathonVersion
```

Comment:
do we wanna put dash in the property name? I think so far we were doing camel casing... ?

btw. this would not happen if we had this encoded as case class as we do for all other models... :-)

Reply:
you're right!